src/transactionsOutcomeParsers/tokenManagementTransactionsOutcomeParser.spec.ts

```diff
@@ -562,4 +592,15 @@ describe("test token management transactions outcome parser", () => {
         assert.equal(outcome[0].nonce, nonce);
         assert.equal(outcome[0].burntQuantity, burntQuantity);
     });
+
+    function b64TopicsToBytes(topics: string[]): Uint8Array[] {
```

Comment:
Could have been `map()` & possibly placed in testutils. Also fine here, as it is.

Reply:
It's now using `map()` and I moved it in `testutils`.